postgres/rdspostgres/rdspostgres.go

```diff
@@ -57,6 +68,9 @@ type Params struct {
 // URLOpener opens RDS PostgreSQL URLs
 // like "rdspostgres://user:password@myinstance.borkxyzzy.us-west-1.rds.amazonaws.com:5432/mydb".
 type URLOpener struct {
+	// CertSource specifies how the opener will obtain authentication information.
+	// CertSource must not be nil.
```

Comment:
If it can't be `nil`, then it shouldn't be `nil` in `RegisterPostgres`. I addressed this in #1685 by using the lazy-creds opener pattern we've been using across the project.